feature/communitylist/src/main/java/com/iteneum/communitylist/presentation/CommunityListViewModel.kt

```diff
@@ -18,26 +18,21 @@ import kotlinx.coroutines.launch
 @HiltViewModel
 class CommunityListViewModel : ViewModel() {
 
+    /* THIS STATE IS GOING TO BE USED WHEN WE NEED TO DO A BIG CHANGE TO THE VIEW  */
     private val _state: MutableStateFlow<CommunityListState> =
         MutableStateFlow(CommunityListState.Loading)
     val state: StateFlow<CommunityListState> = _state
 
-    private val _dataInfo: MutableList<ItemModel> = mutableStateListOf()
-    val dataInfo: MutableList<ItemModel>
-        get() = _dataInfo
+    var myData: MutableList<ItemModel> = mutableStateListOf()
```

Comment:
Could you being more descriptive in the model that should be a domain and in the variable myData, this is useful to wont do a lot of changes in the future

Reply:
Do you mean to put a description for "myData"?

Reply:
I meant that you could be more descriptive with the name of the variable and the model's name

Reply:
Done